src/posts/introducing_lucky_0.15.cr

```diff
@@ -94,6 +94,25 @@ class IntroducingTheLuckyBlog < BasePost
     end
     ```
 
+    ### Fallback routing
+
+    You can think of this like a "catch-all" route. If no other route matches in your application,
+    Lucky will run this as a last try. To use it, just create your action and use `fallback`.
+
+    This can be a great way to have Lucky pass route handling over to your front-end framework!
+
+    ```crystal
+    class VueHandlerAction < BrowserAction
+      fallback do
+        if html?
+          render VueApp::IndexPage
+        else
+          raise Lucky::RouteNotFoundError.new(context)
+        end
+      end
+    end
+    ```
+
```

Comment:
Maybe add a "Learn more in the fallback routing guide" that links to the fallback routing section